boards/arm/nrf5340dk_nrf5340/nrf5340dk_nrf5340_cpuapp.yaml

```diff
@@ -18,4 +18,5 @@ supported:
   - usb_device
   - netif:openthread
   - gpio
+  - spi
```

Comment:
Why not do the same for the `_ns` target?

Reply:
good point, fixed